src/current/cockroachcloud/migrations-page.md

```diff
@@ -138,6 +138,13 @@ If the credentials are valid, they will be added to the [**Credentials** table](
 
 </section>
 
+{{site.data.alerts.callout_info}}
+The Schema Conversion Tool creates the following internal objects when you convert a schema:
+
+- A database prefixed with `_migration_internal_` is created on the {{ site.data.products.cloud }} cluster each time you [retry a schema migration](#retry-the-migration). It does not contain any data apart from the statements in the `_migration_internal_statements` table. When you successfully [migrate a schema](#migrate-the-schema) to the {{ site.data.products.cloud }} cluster, the final `_migration_internal_` database is renamed to your specified database name, and the other `_migration_internal_` databases associated with the schema are removed. `_migration_internal_` databases are also removed when you delete their associated schema from the [**Schemas** table](#schemas-table).
```

Comment:
Maybe worth saying its created on the initial upload and any subsequent retries. Since the first one isn't a "retry"